doc/api/cli.md

```diff
@@ -511,6 +532,17 @@ added: v2.4.0
 
 Track heap object allocations for heap snapshots.
 
+### `--type=type`
```

Comment:
It seems kind of odd that the flag `--type` without any prefixes is dedicated to ESM, is this expected to change when we unflag `--experimental-modules`?

Reply:
There is a conversation going on right now within the modules team about whether or not we want to change the name of the `type` flag. It was meant to mirror `package.type`, which in itself is based on `<script type="module">`.

We may need to fix the docs because the flag requires and argument

Reply:
Yeah, I have to agree that type is far too generic here. This definitely should be changed.

Reply:
There was some discussion of this before, but there was not consensus to change it. (Personally I've love for it to be renamed.)

I believe `--entry-type` was the specific name proposed.

Reply:
How is it dedicated to ESM? `--type=commonjs` says to load a file as CommonJS . . . ?

Reply:
i believe joyee meant the esm loader, not source text modules.

Reply:
I'm going through and updating to `--entry-type` for now, and will punt back to the modules team regarding the change.

@joyeecheung where this effects both loaders is the behavior of `node entry.js`.

While it is very likely the default for node is never going to change folks could guard for the future with `node --entry-type=commonjs entry.js` to ensure they will always get the CJS loader.